src/main/java/org/metricshub/jawk/jrt/JRT.java

```diff
@@ -237,7 +237,7 @@ public static String toAwkStringForOutput(Object o, String ofmt, Locale locale)
 			try {
 				o = new BigDecimal(o.toString()).doubleValue();
 			} catch (NumberFormatException e) {
-				LOG.debug("Failed to parse number", e);
+				LOG.trace("Failed to parse number", e);
```

Comment:
Even this one can be safely removed.
You may need to add a checkstyle tag to prevent it from complaining about the now-empty catch block. 😅